### PR TITLE
Fix displaying on mobile and large desktop

### PR DESCRIPTION
**Pull request by Cyroxin**
@MartinNowak 

Heya! 

Hopefully this repo is still maintained. This has made displaying pages on different screen sizes work better for me, so I am hoping to add this so that I do not have to write it manually to every html page. 

Regards,
Cyroxin

### CODE REVIEW

**Review by MartinNowak — COMMENTED** [review]
Here it says, we can just set the width to 600 to get an minimum width.
https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Viewport_width_and_screen_width
Could you try that on a desktop, seems simpler than the js approach.

File: public/css/style.min.css [hunk 1]
```diff
@@ -1,3 +1 @@
...eight:1.35;letter-spacing:-.08rem}h5{font-size:1.8rem;line-height:1.5;letter-spacing:-.05rem}h6{font-size:1.5rem;line-height:1.6;letter-spacing:0}@media(min-width:550px){h1{font-size:5.0rem}h2{font-size:4.2rem}h3{font-size:3.6rem}h4{font-size:3.0rem}h5{font-size:2.4rem}h6{font-size:1.5rem}}p{margin-top:0}a{color:#1eaedb}a:hover{color:#0fa0ce}.button,button,input[type="submit"],input[type="reset"],input[type="button"]{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:11px;font-weight:600;line-height:38px;letter-spacing:.1rem;text-transform:uppercase;text-decoration:none;white-space:nowrap;background-color:transparent;border-radius:4px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}.button:hover,button:hover,input[type="submit"]:hover,input[type="reset"]:hover,input[type="button"]:hover,.button:focus,button:focus,input[type="submit"]:focus,input[type="reset"]:focus,input[type="button"]:focus{color:#333;border-color:#888;outline:0}.button.button-primary,button.button-primary,input[type="submit"].button-primary,input[type="reset"].button-primary,input[type="button"].button-primary{color:#FFF;background-color:#33c3f0;border-color:#33c3f0}
```
Comment (MartinNowak) [inline public/css/style.min.css:1]: What changed here? Seems weird to edit the minified css.

**Review by MartinNowak — COMMENTED** [review]
You can find the css sources in assets.

File: public/css/style.min.css (hunk 1 above)
Comment (Cyroxin) [reply public/css/style.min.css:1]: It is a bit clearer on my personal repo [commit](https://github.com/Cyroxin/Elembuf/commit/642ec2f68775420604c059c9d966683b51a149e7#diff-872eba2b74eaa10cd9514c3d79a8e7a6R3). These were added:
```` css
#main-contents {
    width: 100%;
} 
.prototype {
    max-width: fit-content;
}
````
Then put into a css [minifier](https://duckduckgo.com/?q=css+minify&ia=answer).

**Comment by Cyroxin** [comment]
> Here it says, we can just set the width to 600 to get an minimum width.
> https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Viewport_width_and_screen_width
> Could you try that on a desktop, seems simpler than the js approach.

I've changed it to a css-only solution. While this doesn't do as well with overflowing, as it has to create a scroll bar for code blocks, it is zoomable, which the script solution wasn't. Another addition was made to the style.min.css:

BEFORE
```` css
pre > code {
    display: block;
    padding: 1rem 1.5rem;
    white-space: pre;
}
````

AFTER
```` css
pre > code {
    display: block;
    padding: 1rem 1.5rem;
    white-space: pre;
    overflow-x: scroll;
}
````

Regards,
Cyroxin

**Comment by Cyroxin** [comment]
I've changed the css to use auto instead of scroll. This seems to work really well for desktop and mobile.

**Comment by Cyroxin** [comment]
This PR is ready @MartinNowak

**Comment by Cyroxin** [comment]
CI seems borked, not sure why. Seems to complain about precompiled assets.

**Comment by Cyroxin** [comment]
@MartinNowak my guess is that the error is due to the minified css being directly modified instead of the assets css.

**Pull request merged**